lib/asker.js

```diff
@@ -315,7 +288,7 @@ Request.prototype.formatTimestamp = function() {
     var timers = this.getTimers();
 
     // @todo let user to use custom format
-    return ['in ', timers.network || '0', '~', timers.total, ' ms'].join('');
+    return 'in ' + (timers.network || '0') + '~' + timers.total + ' ms';
```

Comment:
But the format haven't been changed here! It's about switching from `join('')` to string concatenation.